### ISSUE TITLE
How much memory of GPU is needed to train yolov3

### ISSUE THREAD

**Issue by Ostnie** [qqwweee/keras-yolo3#255]
keep the batchsize as 32, it is too large for 1080Ti, I wonder how much memory shoud be provided for train ? I guess decrease the batchsize will harm the accuracy。l

update: I set the batchsize as 8,and the loss achieves 22 after 80 epoches with earlystopping, and the test MAP is only 6.84%  in voc2012, I set IOU with 0.3 and confidence =0.1 in test.

**Comment by Fingerl** [comment]
hello，can you tell me how to calculate map ?

**Comment by anshan3** [comment]
hello, my GPU is also 1080Ti and i have the same problem. Have you found the solution?

**Comment by leoluopy** [comment]
i use a batch size of 2 , and got a good result in own dataset with mAP 67%

**Comment by anshan3** [comment]
> i use a batch size of 2 , and got a good result in own dataset with mAP 67%

I also use the batch size of 2, after 89 epoches with earlystopping, Do you have the same question?

**Comment by leoluopy** [comment]
> > i use a batch size of 2 , and got a good result in own dataset with mAP 67%
> 
> I also use the batch size of 2, after 89 epoches with earlystopping, Do you have the same question?

 i disabled early stop , and run about 200 epoch , with checking the loss diagram , this is still space for descent. ( early stop can be regarded as regularization)

**Comment by anshan3** [comment]
> > > i use a batch size of 2 , and got a good result in own dataset with mAP 67%
> > 
> > 
> > I also use the batch size of 2, after 89 epoches with earlystopping, Do you have the same question?
> 
> i disabled early stop , and run about 200 epoch , with checking the loss diagram , this is still space for descent. ( early stop can be regarded as regularization)

I see. Thanks a lot!

**Comment by sirius0503** [comment]
@leoluopy : How did you use mAP, I also want to calculate IoU?

**Comment by zhangfx123** [comment]
请问map如何计算的
